### ISSUE TITLE
The entire batch request fails if one of the requested entities does not exist

### ISSUE THREAD

**Issue by baddlan** [DigitalState/Cms#10]
Sample request:
`GET http://api.cms.demo.digitalstate.io/app_dev.php/content?texts[]=portal-title&files[]=some-nonexistent-file-slug`

Response: `404 Not found`

It would be nice to ignore missing entities.

**Comment by mario-digitalstate** [comment]
Fixed c04928c24204ef91d890ecc69a529c6df875c7cc

**Comment by baddlan** [comment]
When fetching multiple (existing) file entities, only one entity appears complete in the response, the reset get `null` values.

Test:
```
http://api.cms.demo.digitalstate.io/app_dev.php/content?texts[]=admin-title&files[]=admin-logo-login&files[]=admin-logo-header&files[]=admin-logo
```

**Comment by mario-digitalstate** [comment]
Fixed b930e75cae0096045e0dda56c1bc9d67b856d228
